.goreleaser.yaml

```diff
@@ -95,6 +95,19 @@ chocolateys:
     api_key: '{{ .Env.CHOCOLATEY_API_KEY }}'
     source_repo: "https://push.chocolatey.org/"
     goamd64: v1
+    
+aurs:
+  - name: fjira-bin
+    homepage: https://github.com/mk-5/fjira
+    description: Fjira - the fuzziest Jira command line tool in the world
+    maintainers:
+      - 'Jonathan Neidel <aur at jneidel dot com>'
```

Comment:
can you add me as a second maintainer here?

- `Mateusz Kulawik <mateusz+aur@mk5.pl>`